### PR TITLE
Support page

### PR DESCRIPTION
**Pull request by jamespohalloran**
Add support page

### CODE REVIEW

File: content/docs/support.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Support
+id: /docs/support/
+last_edited: '2023-08-30T11:40:00.000Z'
+---
+
+## Community support
+
+TinaCMS's front-end is offered as free and open-source. Support is provided via our [Community Discord Server](https://discord.com/invite/zumN63Ybpf), and bugs/feature-requests can be filed in the [TinaCMS GitHub repo](https://github.com/tinacms/tinacms/issues/new/choose) directly.
```
Comment (kldavis4) [inline content/docs/support.md:9]:
```suggestion
TinaCMS's front-end is offered as free and open-source. Support is provided via our [Community Discord Server](https://discord.com/invite/zumN63Ybpf) (Please use the #help channel). Bugs/feature-requests can be filed directly in the [TinaCMS GitHub repo](https://github.com/tinacms/tinacms/issues/new/choose).
```

**Comment by jamespohalloran** [comment]
> @jamespohalloran added a note about verifying a different email address. Also wondering if adding a big Support link at the top of the homepage would be good.

Could nest it under "Resources" maybe? We're a bit limited on real-estate for adding new high-level nav links, but could maybe fit it if we expand the breakpoints

**Comment by kldavis4** [comment]
> > @jamespohalloran added a note about verifying a different email address. Also wondering if adding a big Support link at the top of the homepage would be good.
> 
> Could nest it under "Resources" maybe? We're a bit limited on real-estate for adding new high-level nav links, but could maybe fit it if we expand the breakpoints

yeah, that seems like a good spot

**Comment by jamespohalloran** [comment]
> > > @jamespohalloran added a note about verifying a different email address. Also wondering if adding a big Support link at the top of the homepage would be good.
> > 
> > 
> > Could nest it under "Resources" maybe? We're a bit limited on real-estate for adding new high-level nav links, but could maybe fit it if we expand the breakpoints
> 
> yeah, that seems like a good spot

Updated!

**Comment by kldavis4** [comment]
@jamespohalloran sorry - forgot to add my other suggestions to the review

**Pull request merged**